.rubocop.yml

```diff
@@ -4,7 +4,7 @@ inherit_gem:
   gc_ruboconfig: rubocop.yml
 
 AllCops:
-  TargetRubyVersion: 3.0.1
+  TargetRubyVersion: 2.6
```

Comment:
Is this downgrade only so that rubocop doesn't enforce more style rules?

Reply:
Yep, otherwise it suggests changes that don't work with older versions